mono/metadata/domain.c

```diff
@@ -526,7 +526,6 @@ mono_init_internal (const char *filename, const char *exe_filename, const char *
 #endif
 
 #ifndef HOST_WIN32
-	mono_w32handle_init ();
 	wapi_init ();
 #endif
```

Comment:
this is needed because the profiler might try to create a new handle before having called `mono_init_internal`. I am going to revert that change anyway.